schema/query/fuzz.go

```diff
@@ -0,0 +1,51 @@
+// +build gofuzz
+
+package query
```

Comment:
If you look at tests as documentation, it makes names read as they would when you use the package elsewhere. E.g. you would import the`query` package and use `query.And` over `And` etc., which in some cases can make the names more useful. It also forces you to stick with the public API, which is usually what you want to keep working as is when you do changes.